app/controllers/mixins/actions/vm_actions/resize.rb

```diff
@@ -54,7 +54,12 @@ def resize_vm
                 flavor_id = params['flavor_id']
                 flavor = find_record_with_rbac(Flavor, flavor_id)
                 old_flavor_name = @record.flavor.try(:name) || _("unknown")
-                @record.resize_queue(session[:userid], flavor)
+                # this is always a new request rather than an edit
```

Comment:
I do not think this assumption is true.  Now that this is a request with approval it is possible that the request does not get auto-approved and the admin can edit and modify the value.

Reply:
@gmcculloug OK. I'll need to dig into this again then, but the first time through I wasn't sure what to do about the request_id. Is there something in particular that I need here to make this work, or in the edit case will this already be set to the right value? i.e. do I just delete the "@request_id = nil" line, or do I replace it with something else?

Reply:
Hoping @h-kataria can help here or recommend someone.